src/integration/pages/storyPage/mundo/__snapshots__/canonical.test.js.snap

```diff
@@ -306,7 +306,7 @@ Object {
       "image": Object {
         "@type": "ImageObject",
         "height": 576,
-        "url": "https://news.files.bbci.co.uk/ws/img/logos/og/mundo.png",
+        "url": "http://ichef.test.bbci.co.uk/news/1024/branded_mundo/17D42/production/_110620679_gettyimages-542984918.jpg",
```

Comment:
I'm curious, why do these go from secure to non-secure?

Reply:
thats a good spot, wonder if it is just our localhost setup. will take a look.

Reply:
This has been fixed now.